bigquery/pom.xml

```diff
@@ -197,28 +202,42 @@
                   <include>com.google.http-client:google-http-client-jackson2</include>
                   <include>com.google.oauth-client:google-oauth-client</include>
                   <include>com.google.oauth-client:google-oauth-client-java6</include>
+                  <include>io.grpc</include>
+                  <include>io.netty.intern</include>
                   <include>org.apache.httpcomponents:httpclient</include>
                   <include>org.apache.httpcomponents:httpcore</include>
-                  <!-- needed by http client and versions conflict -->
+                  < needed by http client and versions conflict
                   <include>commons-codec:commons-codec</include>
                 </includes>
-              </artifactSet>
-              <minimizeJar>true</minimizeJar>
+              </artifactSet-->
+              <!--minimizeJar>true</minimizeJar-->
```

Comment:
I don't think that this is good to merge this PR with commented parts of `pom.xml`